src/core/MIGRATION_EXAMPLES.md

```diff
@@ -0,0 +1,255 @@
+all Core services and any services registered by plugins this plugin depends on.
+
+```ts
+new kibana.Plugin({
+  init(server) {
+    const { callWithRequest } = server.plugins.elasticsearch.getCluster('data');
+
+    server.route({
+      path: '/api/my-plugin/my-route',
+      method: 'POST',
+      async handler(req, h) {
+        const results = await callWithRequest(req, 'search', query);
+        return { results };
+      }
+    });
+  }
+});
+
+class Plugin {
+  public setup(core) {
+    const router = core.http.createRouter();
+    router.post(
+      {
+        path: '/api/my-plugin/my-route',
+      },
+      async (context, req, res) => {
+        const results = await context.elasticsearch.dataClient.callAsCurrentUser('search', query);
+        return res.ok({
+          body: { results }
+        });
+      }
+    )
+  }
+}
+```
+
+## Chrome
+
+In the Legacy Platform, the `ui/chrome` import contained APIs for a very wide
+range of features. In the New Platform, some of these APIs have changed or moved
+elsewhere.
+
+| Legacy Platform                                       | New Platform                   | Notes                                                                                                                                          |
+|-------------------------------------------------------|--------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------|
+| `chrome.addBasePath`                                  | `core.http.basePath.prepend`   |                                                                                                                                                |
+| `chrome.breadcrumbs.set`                              | `core.chrome.setBreadcrumbs`   |                                                                                                                                                |
+| `chrome.getUiSettingsClient`                          | `core.uiSettings`              |                                                                                                                                                |
+| `chrome.helpExtension.set`                            | `core.chrome.setHelpExtension` |                                                                                                                                                |
+| `chrome.setVisible`                                   | `core.chrome.setVisible`       |                                                                                                                                                |
+| `chrome.getInjected`                                  | --                             | Not yet available, [comming soon]()                                                                                           |
```

Comment:
It's not entirely true. I remember that we don't want to push plugins to use the deprecated version, but here we have a strong statement that it's not available at all.